src/server/game/DataStores/DBCEnums.h

```diff
@@ -713,7 +713,7 @@ enum ItemBonusType
     ITEM_BONUS_SCALING_STAT_DISTRIBUTION    = 11,
     ITEM_BONUS_DISENCHANT_LOOT_ID           = 12,
     ITEM_BONUS_SCALING_STAT_DISTRIBUTION_2  = 13,
-    ITEM_BONUS_ITEM_LEVEL_OVERRIDE          = 14,
+    ITEM_BONUS_ITEM_LEVEL_OVERRIDE          = 14,                 // Functionality removed in 7.2.0
```

Comment:
The comment is also incorrect - functionality isn't entirely removed, it still makes the client display '+' sign next to item level when viewing tooltip (in quest rewards or dungeon journal loot)

Reply:
I see, so it's only a client-side visual flag now. Should it be renamed or just change the comment?

Reply:
Rename it